rfc6962-bis.xml

```diff
@@ -375,13 +375,13 @@ REDACT(label) = prefix || BASE32(index || LABELHASH(keyid || label))</artwork>
             <spanx style="verb">prefix</spanx> is the "?" character (ASCII value 63).
           </t>
           <t>
-            <spanx style="verb">||</spanx> denotes concatenation.
+            <spanx style="verb">||</spanx> denotes length-encoded concatenation. Each concatenated value is preceded by a single byte that contains the length (in bytes) of that value.
```

Comment:
Sorry, was thinking of the entire name. But where do we say the SAN must be a DNS name?